lib/basedriver/commands/session.js

```diff
@@ -2,16 +2,26 @@ import _ from 'lodash';
 import log from '../logger';
 import { errors } from '../../mjsonwp';
 import UUID from 'uuid-js';
+import { processCapabilities } from '../capabilities';
 
 let commands = {};
 
-commands.createSession = async function (caps) {
+commands.createSession = async function (desiredCapabilities, requiredCaps, capabilities) {
```

Comment:
looks like we're breaking the API here; let's remember to release this as a major rev